File: cloud-storage-spring-java/src/main/java/com/github/sevtech/cloud/storage/spring/property/gcloud/GoogleCloudStorageProperties.java
package com.github.sevtech.cloud.storage.spring.property.gcloud;

import lombok.Getter;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;

@Getter
@Setter
@ConfigurationProperties(prefix = "gcp.storage")
public class GoogleCloudStorageProperties {

	private String keyfile;

}

File: cloud-storage-spring-kotlin/src/main/kotlin/com/github/migangqui/cloud/storage/spring/property/GoogleCloudStorageProperties.kt
package com.github.migangqui.cloud.storage.spring.property

import org.springframework.core.env.Environment

class GoogleCloudStorageProperties(private val env: Environment) {

    private val KEYFILE_LOCATION = "gcp.storage.keyfile"

    val keyfileLocation: String?
        get() = env.getProperty(KEYFILE_LOCATION)

}